src/services/userService.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+let users = [];
```

Comment:
You can use const in this place. 
I see you manipulate with an array, easy way to use const and avoid reassigned.